File: models/person.py
from random import random

class Person:

    # человек, носитель истории, субъект, объект, тот кто управляет историей и кем история управляет

    def __init__(self, name, uuid):
        self._stories = list()
        self._name = name
        self._uuid = uuid

    def tell_random(self):
        n = round(random()*1)
        r = self._stories[n-1].tell()
        return r

    def uuid(self):
        return self._uuid

    def name(self):
        return self._name

File: Noosphere/models/Person.cs
using System;

namespace BrocaZone.models;

public class Person
{
    public Person(string id, string name)
        {
            Id = id;
            Name = name;
            Stories = new List<Story>();
        }

    public string Id { get; set; }

    public string Name { get; set; }

    public List<Story> Stories { get; set; }
}
